staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -718,14 +727,16 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 			}
 		}
 
-		var scaleSpec *apiextensions.CustomResourceSubresourceScale
+		var scaleSpec = &apiextensionsinternal.CustomResourceSubresourceScale{}
```

Comment:
this now is always non-nil. Has that side-effects?

Reply:
https://github.com/kubernetes/kubernetes/pull/84005#discussion_r352060472